Mobile/starter_project_dartsmiths/lib/features/onboarding/onboardingLowerLayout.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:flutter/material.dart';
+class LandingPageLowerImage extends StatelessWidget {

```

Comment:
[nit] should we have one line space between imports and the rest?

Reply:
I would also like the name changed. there is no landing page in the app it is onboarding page and the name can be more formal also.